bower_components/aura/lib/ext/mediator.js

```diff
@@ -92,7 +91,6 @@ define('aura/ext/mediator', function () {
        * @param {Object} payload    Payload emitted
        */
       app.sandbox.emit = function () {
-        if (this.stopped) { return; }
```

Comment:
As said, this line is also missing in the original library, so it was probably removed there as well: https://github.com/aurajs/aura/blob/master/lib/ext/mediator.js#L93

It doesn't look like the component is lost, otherwise it would not work, I guess.